br/pkg/task/restore.go

```diff
@@ -271,16 +271,16 @@ func CheckNewCollationEnable(
 	backupNewCollationEnable string,
 	g glue.Glue,
 	storage kv.Storage,
-	CheckRequirements bool,
+	SkipCollactionCheck bool,
 ) error {
 	if backupNewCollationEnable == "" {
-		if CheckRequirements {
+		if !SkipCollactionCheck {
```

Comment:
Should we skip check at here too?
<img width="695" alt="image" src="https://user-images.githubusercontent.com/36239017/160787085-85883025-9c32-41f2-8aa0-1b99bc1795bc.png">

Reply:
> Should we skip check at here too? <img alt="image" width="695" src="https://user-images.githubusercontent.com/36239017/160787085-85883025-9c32-41f2-8aa0-1b99bc1795bc.png">

Can't skip when `NewCollactionEnable` config is different between backupmeta and restore cluster. It will case data  error.